examples/04-advanced/14-incremental_evaluation.py

```diff
@@ -0,0 +1,34 @@
+"""
+.. _ref_incremental_evaluation:
+"""
+
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+
+def scop_from_ds(ds):
```

Comment:
@ansys-akarcher the fact that you had to add this helper reflects that this is missing in core I think. 
`scoping_on_all_time_freqs` taking already a model, let's improve it to take a data_sources alors
![image](https://github.com/pyansys/pydpf-core/assets/80476446/74d55869-b572-42ef-a8cd-1cdbba712940)